Recursive/PalindromePartitioning.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Problem Statement: Given a string s, partition s such that every substring of the partition is a palindrome. Return all possible palindrome partitioning of s.
+ * what is palindrome partitioning?
+ * - Palindrome partitioning means, partitioning a string into substrings such that every substring is a palindrome.
+ * Reference to know more about palindrome partitioning:
+ * - https://www.cs.columbia.edu/~sedwards/classes/2021/4995-fall/proposals/Palindrome.pdf
+ */
+
+// class PalindromePartitioning {
```

Comment:
Please remove this outcommented code.

Reply:
I already deleted that in my third commit that i made "clean up code"

I don't know if i am getting you well, but please check out updates of my third commit

Reply:
oops yeah sorry, was reviewing the second commit there.